pkgs/development/ocaml-modules/repr/default.nix

```diff
@@ -1,28 +1,31 @@
 {
   lib,
   buildDunePackage,
-  fetchFromGitHub,
+  fetchurl,
+  ocaml,
   base64,
   either,
   fmt,
   jsonm,
   uutf,
   optint,
+  version ? if lib.versionAtLeast ocaml.version "5.3" then "0.8.0" else "0.7.0",
```

Comment:
I take it this is due to the differing `ppxlib` versions between >= 5.3 and < 5.3?

Reply:
This may be clearer/easier to maintain?

Reply:
Updated.